pillar/edx/ansible_vars/residential.sls

```diff
@@ -167,6 +167,30 @@ edx:
         readPreference: "nearest"
       SOCIAL_AUTH_SAML_SP_PRIVATE_KEY: __vault__::secret-residential/{{ environment }}/{{ purpose }}/saml-sp-cert>data>key
       SOCIAL_AUTH_SAML_SP_PUBLIC_CERT: __vault__::secret-residential/{{ environment }}/{{ purpose }}/saml-sp-cert>data>value
+      PROCTORING_SETTINGS:
+        MUST_BE_VERIFIED_TRACK: false
+      PROCTORING_BACKENDS:
+        DEFAULT: "proctortrack"
+        proctortrack:
+          client_id: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>client_id
+          client_secret: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>client_secret
+          base_url: __vault__::secret-{{ business_unit }}/{{ environment }}/proctortrack>data>base_url
+      JWT_AUTH:
+        JWT_AUDIENCE: 'pt_openedx_key'
+      DEFAULT_JWT_ISSUER:
+        AUDIENCE: "https://{{ purpose_data.domains.lms }}"
+      JWT_AUTH:
+        JWT_AUDIENCE: "pt_openedx_key"
+        JWT_ISSUER: "https://{{ purpose_data.domains.lms }}/oauth2"
+        JWT_ISSUERS:
+          - AUDIENCE: "pt_openedx_key"
```

Comment:
Should these all be separate elements in a list, or is this a list of dicts?

Reply:
This one's weird, cause that's how it is in the `lms.yml` file so guess when salt renders it's gonna remove the hyphen

Reply:
That's how it was setup on QA before